### PR TITLE
Home section

### PR DESCRIPTION
**Pull request by heba-elsafty**

### CODE REVIEW

File: src/assets/styles/components/_banner.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.banner {
```
Comment (Sohype-Khaled) [inline src/assets/styles/components/_banner.scss:1]: remove grid system and use flex
use clip path
height in .banner only 
base color :default
create classes for all basic colors : primary, secondary accent success warning info danger

File: src/assets/styles/components/_preference-survey.scss [hunk 2]
```diff
@@ -0,0 +1,106 @@
+.preference-survey {
```
Comment (Sohype-Khaled) [inline src/assets/styles/components/_preference-survey.scss:1]: rename service-excerpt
dont nest the modifiers name

File: src/assets/styles/components/_banner.scss (hunk 1 above)
Comment (heba-elsafty) [reply src/assets/styles/components/_banner.scss:1]: done

File: src/assets/styles/components/_preference-survey.scss (hunk 2 above)
Comment (heba-elsafty) [reply src/assets/styles/components/_preference-survey.scss]: done

**Pull request merged**